### PR TITLE
[JSD Selenium] DCA-878: browse customers page

### PR DESCRIPTION
**Pull request by mmizin**
https://bulldog.internal.atlassian.com/browse/DCA-878

### CODE REVIEW

File: app/selenium_ui/jsd/modules_agents.py [hunk 1]
```diff
@@ -66,3 +67,13 @@ def measure():
...
         browse_projects_page.wait_for_page_loaded()
     measure()
+
+
+def browse_project_customers_page(webdriver, datasets):
+    browse_customers = BrowseCustomers(webdriver)
+
+    @print_timing('selenium_browse_project_customers_page')
+    def measure():
+        browse_customers.go_to_url(UrlManager(project_key=datasets['project_key']).browse_project_customers_page())
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/jsd/modules_agents.py:77]: Please create an appropriate method in the Page class. So it should be like `CustomersPage` with argument as project_key, `customers_page = Customers(project_key=...)` and `customers_page.go_to`

Comment (mmizin) [reply app/selenium_ui/jsd/modules_agents.py]: Updated

File: app/selenium_ui/jsd/pages/agent_pages.py [hunk 2]
```diff
@@ -46,3 +47,13 @@ class BrowseProjects(BasePage):
 
     def wait_for_page_loaded(self):
         self.wait_until_visible(BrowseProjectsLocators.page_title)
+
+
+class BrowseCustomers(BasePage):
+
+    def __init__(self, driver, project_key=None):
+        BasePage.__init__(self, driver)
+        self.page_url = UrlManager(project_key=project_key).browse_project_customers_page_url()
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/jsd/pages/agent_pages.py:56]: Let's follow code style convention for page objects as we have in already developed solutions for Jira, etc.
`self.url_manager = UrlManager(project_key=project_key), self.url_manager.browse_project_customers_page_url()`

File: app/selenium_ui/jsd/modules_agents.py [hunk 3]
```diff
@@ -66,3 +67,13 @@ def measure():
         browse_projects_page.go_to()
         browse_projects_page.wait_for_page_loaded()
     measure()
+
+
+def browse_project_customers_page(webdriver, datasets):
+    browse_customers = BrowseCustomers(webdriver, project_key=datasets['project_key'])
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/jsd/modules_agents.py:73]: `browse_customers_page`

File: app/selenium_ui/jsd/pages/agent_pages.py (hunk 2 above)
Comment (mmizin) [reply app/selenium_ui/jsd/pages/agent_pages.py:56]: Just url_manager not self.url_manager ?

File: app/selenium_ui/jsd/modules_agents.py (hunk 3 above)
Comment (mmizin) [reply app/selenium_ui/jsd/modules_agents.py:73]: Updated

**Review by SergeyMoroz0703 — APPROVED** [review]

**Review by ometelytsia — APPROVED** [review]

**Pull request merged**